utils/shared.js

```diff
@@ -1260,6 +1265,22 @@ const validateCancerOccurrence = (cancerSitesObject) => {
     return { error: hasError, message: hasError ? otherCancerSiteErrorMessage : '', data: [] };
 }
 
+/**
+ * Rules: if vitalStatusCategorical is 'alive' at chart review (114227122: 337516613), participant must be aware of diagnosis (844209241: 353358909). Else, block API request.
+ * If vitalStatusCategorical is 'dead' or 'unknown' (114227122: 646675764 or 178420302), participant awareness can be yes, no, or unknown (844209241: 353358909 or 104430631 or 178420302).
+ * @param {number} vitalStatusCategorical - the participant's vital status (conceptID).
+ * @param {number} participantDiagnosisAwareness - the participant's awareness of diagnosis (conceptID).
```

Comment:
NIT: Should we define `return`

Reply:
Good gatch, thanks. Updated in this function and `validateCancerOccurrence`